tasks/python3/processVectorData.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+from optparse import OptionParser
+import os
+import sys
+import json
+import shutil
+
+prog = os.path.basename(__file__)
+base_dir = os.path.join(os.path.dirname(__file__), "..")
+version = "3.0.0"
+help_description = """\
+Converts vector data to JSON files
+"""
+
+parser = OptionParser(usage="Usage: %s <config_file> <vectordata_input_dir> <output_dir>" % prog,
+                      version="%s version %s" % (prog, version),
+                      epilog=help_description)
+
+(options, args) = parser.parse_args()
+if len(args) != 3:
+    parser.error("Invalid number of arguments")
+
+config_file = args[0]
+vectordata_input_dir = args[1]
+output_dir = args[2]
+
+with open(config_file) as fp:
+    config = json.load(fp)
+
+def copy_file(file):
+    input_file = os.path.join(root, file)
+    if input_file.endswith('.json'):
+        response_data = {}
+        vector_layer_filename = file;
+        vector_layer_id = vector_layer_filename.split(".", 1)[0]
```

Comment:
I have an open PR that changes this line; It probably makes sense to get your work in first and I will rebase my PR on top of it and add the change to the python3 directory. https://github.com/nasa-gibs/worldview/pull/2013